tests/test_layout.py

```diff
@@ -39,3 +39,72 @@ def test_that_product_older_versions_are_ordered_correctly(self, mozwebqa):
         csp = CrashStatsHomePage(mozwebqa)
 
         Assert.is_sorted_descending(csp.header.other_versions)
+
+
+class TestSuperSearchLayout:
+
+    @pytest.mark.nondestructive
+    def test_super_search_page_is_loaded(self, mozwebqa):
+        csp = CrashStatsHomePage(mozwebqa)
+        cs_super = csp.header.click_super_search()
+
+        Assert.true(cs_super.is_the_current_page)
+
+    @pytest.mark.nondestructive
+    def test_default_fields_for_firefox(self, mozwebqa):
+        csp = CrashStatsHomePage(mozwebqa)
+        cs_super = csp.header.click_super_search()
+        cs_super.open_url('/search/?product=Firefox')
```

Comment:
This is a bit odd. You are starting the test by clicking on something in the header, but that becomes irrelevant if you simply navigate to a different page in the browser at this line. Is there a way to navigate to this page using the Crash Stats UI?